Scripts/ParseEmailFiles/parse_email_files_test.py

```diff
@@ -391,3 +391,17 @@ def test_eml_contains_base64_encoded_eml(mocker, email_file):
 
     assert results[0]['EntryContext']['Email'][1]["Subject"] == 'test - inner attachment eml'
     assert results[0]['EntryContext']['Email'][1]['Depth'] == 1
+
+
+def test_msg_headers_map():
+    email_data, ignore = handle_msg('test_data/utf_subject.msg', 'utf_subject.msg')
+    assert 'This is a test email.' in email_data['Text']
```

Comment:
we usually encounters decoding issues with this script
please add to your tests non ascii characters

Reply:
@Itay4 If I understand you correctly, this email is also being tested for such thing at `test_msg_utf_encoded_subject`.
Is it enough?

Reply:
lets verify `Subject` is as expected to this test as well